### ISSUE TITLE
Suggestion of similar navbar

### ISSUE THREAD

**Issue by Mizano1** [jamieSuchodolski/ProgBlack-Group-Project-wk4#2]
i suggest using a different nav bar 

<nav class ="navbar navbar-expand-xl navbar-dark bg-dark">
            <a class ="navbar-brand" href ="#"> Application of computing </a>
            <button class ="navbar-toggler" type ="button" data-toggle ="collapse" data-target ="#colNav">
            <span class ="navbar-toggler-icon"></span>
            </button>
            <div class ="collapse navbar-collapse" id ="colNav">
            <ul class ="navbar-nav">
            <li class ="nav-item">
            <a class ="nav-link" href="#"> Page two</a>
            </li>
            <li class ="nav-item">
            <a class ="nav-link" href ="#"> Page three</a>
            </li>
            <li class ="nav-item">
            <a class ="nav-link" href ="#"> Page four</a>
            </li>
            </ul>
            </div>
        </nav>

**Comment by Mizano1** [comment]
[navbar.txt](https://github.com/jamieSuchodolski/ProgBlack-Group-Project-wk4/files/9914703/navbar.txt)

**Comment by jamieSuchodolski** [comment]
Yeah that's fine. I have however added a margin to make it look neater. I've also added a blank list element at the end of the navbar so that each link appears in the same format (as there is no reason for page 4 to look different). Also, on blackboard it was specified that we needed an index page, which will be the page titled applications of computing, hence why there is now a page 1 link as well. Hope that's ok. I've attached my version of the navbar here
[navbar (1).txt](https://github.com/jamieSuchodolski/ProgBlack-Group-Project-wk4/files/9922691/navbar.1.txt)
